sql/core/src/main/scala/org/apache/spark/sql/catalog/Catalog.scala

```diff
@@ -270,9 +270,10 @@ abstract class Catalog {
    * tied to any databases, i.e. we can't use `db1.view1` to reference a local temporary view.
    *
```

Comment:
can you add a line saying the return type was unit in Spark 2.0, but changed to boolean in Spark 2.1?